README.md

```diff
@@ -36,17 +36,43 @@ or warning it is not installed, also it will create the gemset.
 rvm install ruby-2.4.0
 ```
 
-##### Database creation
+#### Database setup.
 ```
-rais db:setup
+rails db:setup
 ```
 
-##### Database initialization
+#### Install bundler gem at global gemset.
+
+```
+rvm @global do gem install bundler
+```
+
+#### Running falae
+
+Installing gems:
+
 ```
-rails db:seed
+bundle install
 ```
+Set FALAE_IMAGES_PATH environment variable, which should points to a directory where pictograms and users images will be stored. It should not set this variable to the app public directory.
 
-##### System dependencies
+```
+export FALAE_IMAGES_PATH=<dir-name>
+```
+
+Running the application
+
+```
+rails server
+```
+
+There is a rails task to populate the database with some pictogram samples for development purposes.
```

Comment:
I'd change this paragraph to:  `For development purposes, there is a useful rails task..`